resources/bin/crafter.sh

```diff
@@ -497,9 +497,9 @@ function stopTomcat() {
 
 
 function startMongoDB(){
-  echo "------------------------------------------------------------"
+  echo "------------------------------------------------------------------------"
   echo "Starting MongoDB"
-  echo "------------------------------------------------------------"
+  echo "------------------------------------------------------------------------"
   if [ ! -s "$MONGODB_PID" ]; then
```

Comment:
How would this work if someone has an independent MongoDB?
Also, if there is no MongoDB DB folder, isn't that a red flag that means the DB is elsewhere?

Reply:
yeah backup and restore doesn't work if it's an external/independent MongoDB. same thing for Maria. we probably need to create another ticket to find a way to backup in that way.

Reply:
@avasquez614 Please create these tickets now.